programs/compressed-token/src/process_transfer.rs

```diff
@@ -456,9 +479,13 @@ pub mod transfer_sdk {
         leaf_indices: &[u32],
         proof: &CompressedProof,
         input_token_data: &[crate::TokenData],
+        mint: Pubkey,
```

Comment:
FYI: we have to pass mint separately because in case of spl-compress, input_token_data is empty